app/scripts/modules/core/src/overrideRegistry/Overrides.tsx

```diff
@@ -61,3 +61,14 @@ export function Overrides(key: string, cloudProvider?: string) {
     overrideRegistrationQueue.register(targetComponent, key, cloudProvider);
   };
 }
+
+/**
+ * This should be applied to a React Function Component, since they cannot utilize decorators.
```

Comment:
```suggestion

/**
 * Registers a component as a replacement for some other component.
 *
 * This should be applied to a React Function Component, since they cannot utilize decorators.
```